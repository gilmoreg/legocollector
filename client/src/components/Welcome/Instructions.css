.Instructions {
    position: fixed;
    bottom: 110px;
    right: 25px;
    border-radius: 5px;
    background-color: rgba(255,246,229,0.7);
    padding: 1rem;
}

.Instructions .plus {
    display: inline-block;
    border: 0.5rem solid #FF7F66;
    border-radius: 100%;
    background-color: #FF7F66;
    color: white;
    width: 2rem;
    height: 2rem;
    font-size: 14px;
}