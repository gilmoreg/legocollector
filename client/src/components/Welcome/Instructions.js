import React from 'react';
import styled from 'styled-components';

const StyledInstructions = styled.div`
  position: fixed;
  bottom: 110px;
  right: 25px;
  border-radius: 5px;
  background-color: rgba(255,246,229,0.7);
  padding: 1rem;
  z-index: -1;
`;

const StyledPlus = styled.div`
  display: inline-block;
  border: 0.5rem solid #FF7F66;
  border-radius: 100%;
  background-color: #FF7F66;
  color: white;
  width: 2rem;
  height: 2rem;
  font-size: 14px;
`;

const Instructions = () => (
  <StyledInstructions>
    Click on the <StyledPlus>+</StyledPlus> to start watching sets!
  </StyledInstructions>
);

export default Instructions;
