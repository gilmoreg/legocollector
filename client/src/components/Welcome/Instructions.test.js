import React from 'react';
import { shallow } from 'enzyme';
import Instructions from './Instructions';

it('renders without crashing', () => {
  const wrapper = shallow(
    <Instructions />,
  );
  expect(wrapper).toMatchSnapshot();
});
