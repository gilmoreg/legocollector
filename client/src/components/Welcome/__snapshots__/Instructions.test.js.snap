// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Instructions />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "Click on the ",
        <div
          className="plus"
        >
          +
        </div>,
        " to start watching sets!",
      ],
      "className": "Instructions",
    },
    "ref": null,
    "rendered": Array [
      "Click on the ",
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "+",
          "className": "plus",
        },
        "ref": null,
        "rendered": "+",
        "type": "div",
      },
      " to start watching sets!",
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "Click on the ",
          <div
            className="plus"
          >
            +
          </div>,
          " to start watching sets!",
        ],
        "className": "Instructions",
      },
      "ref": null,
      "rendered": Array [
        "Click on the ",
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "+",
            "className": "plus",
          },
          "ref": null,
          "rendered": "+",
          "type": "div",
        },
        " to start watching sets!",
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
